tool/tsh/common/device.go

```diff
@@ -147,3 +158,26 @@ func (c *deviceKeygetCommand) run(cf *CLIConf) error {
 	fmt.Printf("DeviceCredential %s\n", val)
 	return nil
 }
+
+type deviceActivateCredentialCommand struct {
+	*kingpin.CmdClause
+	encryptedCredential       string
+	encryptedCredentialSecret string
+}
+
+func (c *deviceActivateCredentialCommand) run(cf *CLIConf) error {
+	err := dtnative.HandleTPMActivateCredential(
+		c.encryptedCredential, c.encryptedCredentialSecret,
+	)
+	if cf.Debug {
+		if err != nil {
+			// On error, wait for user input before executing. This is because this
+			// opens in a second window. If we return the error immediately, then
+			// this window closes before the user can inspect it.
+			log.WithError(err).Debug("An error occurred during credential activation. Press enter to close this window.")
```

Comment:
This error log is a good feedback for users. Any reason we want to gate it behind debug flag? (referring to above `if cf.Debug {`)

Reply:
So with `tsh`, no logging is emitted unless `-d` is provided. With this situation, we have some weirdness in that this code is exited in a window that will close as soon as `tsh` exits. So even if logs are emitted, they disappear before the user can view it.

The idea here is:
- When `-d` is provided to the parent `tsh` instance, it's also passed to the child
- When the child receives `-d` it waits for user input before exiting, this lets them view the logs.
- When `-d` isn't provided, we just close the child immediately and the parent catches the non-zero status code.

Reply:
Noted. Thanks for explaining.